src/common/com/intellij/plugins/haxe/model/HaxeProjectModel.java

```diff
@@ -30,8 +30,6 @@
 import java.util.List;
```

Comment:
For a future change:  Can we please rename this class to IdeaProjectModel?  (And maybe put it in the .../ide subdirectory?)  This has almost nothing to do with the Haxe language and has everything to do with IDEA.  When I think of what the models do (and those in this subdirectory, in particular), I think of the Haxe language and tool kit.  In my mind a HaxeProjectModel should model the Haxe project files (.hxml, .nmml, .xml), not the reflection/copy that is in the IDEA project.  (Plus, I _also_ wrote a HaxeProjectModel -- that I've had to rename HaxeProjectFileModel :P -- which does model those files.)